<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QPushButton" name="btn_insertar">
      <property name="text">
       <string>Insertar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_modificar">
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_eliminar">
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Fecha de inicio</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Numero de indentificacion</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="5">
     <widget class="QLineEdit" name="valor_direccion_reparto"/>
    </item>
    <item row="2" column="3" colspan="4">
     <widget class="QDateEdit" name="valor_fecha"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Direccion</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLineEdit" name="valor_cantidad_metro_cuadrados"/>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Tipo de Techo</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Reparto</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="5">
     <widget class="QLineEdit" name="valor_direccion_numero"/>
    </item>
    <item row="0" column="6">
     <widget class="QLineEdit" name="valor_numero_identificacion"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Numero</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Calle</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="6">
     <widget class="QLineEdit" name="valor_direccion_calle"/>
    </item>
    <item row="1" column="5" colspan="2">
     <widget class="QLineEdit" name="valor_estado_terminacion"/>
    </item>
    <item row="3" column="0" colspan="6">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Cant. Metros Cuadrados</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Estado de terminacion</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="3">
     <widget class="QComboBox" name="valor_tipo_de_techo">
      <item>
       <property name="text">
        <string>placa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tejas de zinc</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tejas de asbesto cemento</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tejas de arcilla</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Cantidad de cuartos</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="valor_cantidad_cuartos"/>
    </item>
    <item row="11" column="5">
     <widget class="QCheckBox" name="valor_chochera">
      <property name="text">
       <string>Cochera</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tabla_viviendas_ind">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
